### PR TITLE
Improve uniform-buffers.html

### PR DESCRIPTION
**Pull request by zhenyao**
Test named uniform blocks with instance array.

### CODE REVIEW

File: sdk/tests/conformance2/buffers/uniform-buffers.html [hunk 1]
```diff
@@ -272,7 +297,102 @@
...
+    uboFloatView[offset / Float32Array.BYTES_PER_ELEMENT +
+                 uniformOffsets[2] / Float32Array.BYTES_PER_ELEMENT] = 1.0; // Blue
+
+    b1 = gl.createBuffer();
+    gl.bindBuffer(gl.UNIFORM_BUFFER, b1);
+    gl.bufferData(gl.UNIFORM_BUFFER, uboArray, gl.DYNAMIC_DRAW);
+    wtu.glErrorShouldBe(gl, gl.NO_ERROR, "should be able to set UBO data with no errors");
+
+    gl.uniformBlockBinding(program, blockIndex[0], 0);
+    gl.bindBufferRange(gl.UNIFORM_BUFFER, blockIndex[0], b1, 0, blockSize[0]);
```
Comment (toji) [inline sdk/tests/conformance2/buffers/uniform-buffers.html]: `blockIndex[0]` should be `0`

**Comment by toji** [comment]
LGTM after one fix. Feel free to merge when done.

File: sdk/tests/conformance2/buffers/uniform-buffers.html (hunk 1 above)
Comment (zhenyao) [inline sdk/tests/conformance2/buffers/uniform-buffers.html]: Done.  Actually to avoid confusion, I changed the bindings to be 2 and 3.

**Comment by kenrussell** [comment]
Great improvement. LGTM after the fact.

**Pull request merged**